### PR TITLE
Add select all functionality for experiment index page

### PR DESCRIPTION
**Pull request by UAMS16**
image
image

### CODE REVIEW

File: cegs_portal/search/static/search/js/experiment_list/drag_drop.js [hunk 1]
```diff
@@ -116,15 +116,29 @@ function experimentListItemText(name, accession) {
     return `<span class="experiment-list-item" data-accession="${accession}"><span class="exp-name">${name}</span> <span class="accession-id">${accession}</span></span>`;
 }
 
+function selectAllExperiments() {
+    let selectExperiments = document.getElementsByClassName("select-experiment");
+    for (let experiment of selectExperiments) {
+        experiment.click();
+        addToExperimentList(experimentListItemText(experiment.dataset.name, experiment.dataset.accession));
```
Comment (tncowart) [inline cegs_portal/search/static/search/js/experiment_list/drag_drop.js:123]: Do you need both of these? Or will just one work?

Comment (UAMS16) [reply cegs_portal/search/static/search/js/experiment_list/drag_drop.js:123]: Need both to work correctly

Comment (tncowart) [reply cegs_portal/search/static/search/js/experiment_list/drag_drop.js:123]: Are you _sure_? What have you tried?

Comment (UAMS16) [reply cegs_portal/search/static/search/js/experiment_list/drag_drop.js:123]: Are you referring to 
image

Comment (tncowart) [reply cegs_portal/search/static/search/js/experiment_list/drag_drop.js:123]: I'm referring to the two lines I highlighted

**Review by tncowart — APPROVED** [review]
looks good!

**Pull request merged**